src/SprykerEco/Zed/AmazonPay/Dependency/Facade/AmazonPayToShipmentBridge.php

```diff
@@ -31,6 +31,21 @@ public function __construct($shipmentFacade)
      */
     public function getAvailableShipmentMethods(QuoteTransfer $quoteTransfer)
     {
+
+        if (method_exists($this->shipmentFacade, 'getAvailableMethodsByShipment') === true) {
+
+            $shipmentMethodsCollectionTransfer = $this->shipmentFacade->getAvailableMethodsByShipment($quoteTransfer);
+
+            if ($shipmentMethodsCollectionTransfer->getShipmentMethods()->count() > 1) {
+                throw new \Exception('Split shipping is not supported');
```

Comment:
Please create `UnsupportedFeatureException` class. In general, we always create a meaningful exception and don't throw generic ones.

Reply:
In these cases it is not a useful exception to "catch" etc. So a custom one can be overhead. A RuntimeException at least would suffice. I agree that generic Exception should always be avoided.